navigation/AppNavigator.js

```diff
@@ -1,11 +1,13 @@
-import React from 'react'
-import { createAppContainer, createSwitchNavigator, createBottomTabNavigator } from 'react-navigation'
-import MainTabNavigator from './MainTabNavigator'
-import LoginScreen from '../screens/LoginScreen'
-import RegisterScreen from '../screens/RegisterScreen'
-import { createStackNavigator } from 'react-navigation-stack'
-import { Platform, StatusBar } from 'react-native'
-import CreatePaylist from '../screens/CreatePaylist'
+import React from 'react';
```

Comment:
@offler9 try to remove semi column here, maybe you can check your `vscode` linting and make sure to use our `eslint` rule